lib/Pherkin/Extension/Weasel.pm

```diff
@@ -371,6 +373,13 @@ sub _save_screenshot {
         warn "Couldn't open screenshot image '$img_name': $!";
     }
 
+    my $img = Imager->new(); #  Empty image (size is 0 by 0)
+    $img->open(file=>$path_img_name,type=>'png'); # Read image from file
+    # If image is blank
+    if ($img->getcolorcount() == 1 ) {
```

Comment:
Instead of adding an extra dependency, can't we simply check that the result of "get_current_url()" in the Selenium driver is empty? That way, we know there hasn't been any navigation yet.